elide-core/src/main/java/com/yahoo/elide/core/filter/Operator.java

```diff
@@ -350,6 +366,25 @@ public static Operator fromString(final String string) {
         };
     }
 
+    private static <T> Predicate<T> hasMember(String field, List<Object> values, RequestScope requestScope) {
+        return (T entity) -> {
+            if (values.size() != 1) {
+                throw new InvalidPredicateException("HasMember can only take one argument");
+            }
+            Object val = getFieldValue(entity, field, requestScope);
+            String filterStr = CoerceUtil.coerce(values.get(0), String.class);
```

Comment:
coerce into val.getClass().  We probably want to support lists of numbers too.

Reply:
Object `val` will be an instance of Collection. To get the parameterized type of collection, I think we need to iterate over the path and get paremeterizedType from entityDict. I am not sure how to get parameterized type from Collection object

Reply:
You are right, we'll need to iterate over it.  
Another alternative is to convert field into a Path object in Elide.  That will do all the type conversion for you.